common/virtualmachineconstants.go

```diff
@@ -32,7 +32,7 @@ type VmSize struct {
 	GpuName       string
 	MemoryMB      int
 	GpuAssignMode GpuAssignType
-	GpuList       []*pb.Gpu
+	GpuList       []*pb.VirtualMachineGPU
```

Comment:
Is this a breaking change?

Reply:
It's the `.go` file so we don't have to worry about marshaling/unmarshaling issues. Although since this gets used in `moc-pkg`, it will definitely break the current code. I have a `moc-pkg` PR to use this new object.

https://github.com/microsoft/moc-pkg/pull/502